dns-records.go

```diff
@@ -42,25 +44,29 @@ func (cmd *DnsRecords) Run(c *cli.Context) error {
 
 // Get the records from DNSDock and process/return them
 func (cmd *DnsRecords) LoadRecords() ([]map[string]interface{}, error) {
-	if response, httpErr := http.Get("http://dnsdock.outrigger.vm/services"); httpErr == nil && response.StatusCode == 200 {
-		defer response.Body.Close()
-		if body, readErr := ioutil.ReadAll(response.Body); readErr == nil {
-			if js, jsonErr := simplejson.NewJson(body); jsonErr == nil {
-				dnsdockMap, _ := js.Map()
-				records := []map[string]interface{}{}
-				for id, value := range dnsdockMap {
-					record := value.(map[string]interface{})
-					record["Id"] = id
-					records = append(records, record)
+	if ip, ipErr := exec.Command("docker", "inspect", "--format", "{{.NetworkSettings.IPAddress}}", "dnsdock").Output(); ipErr == nil {
```

Comment:
We only need it in one place for now, so lets leave it. If we need it again we can undertake a larger refactoring, or decide if we need to move back to compilation with `cgo`